libcxx/test/std/utilities/expected/expected.void/observers/has_value.pass.cpp

```diff
@@ -43,6 +44,16 @@ constexpr bool test() {
     assert(!e.has_value());
   }
 
+  // See comments of the corresponding test in "expected.expected".
```

Comment:
Can you use a full path to the file where these comments live?

Reply:
done!